com.unity.cinemachine/Editor/Editors/CinemachineConfiner2DEditor.cs

```diff
@@ -1,31 +1,39 @@
+#if CINEMACHINE_PHYSICS_2D
+
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEditor;
 
-#if CINEMACHINE_PHYSICS_2D
 namespace Cinemachine.Editor
 {
     [CustomEditor(typeof(CinemachineConfiner2D))]
     [CanEditMultipleObjects]
     class CinemachineConfiner2DEditor : BaseEditor<CinemachineConfiner2D>
     {
         SerializedProperty m_MaxWindowSizeProperty;
-        GUIContent m_ComputeSkeletonLabel = new GUIContent(
+        GUIContent m_ComputeSkeletonLabel = new(
             "Oversize Window", "If enabled, the confiner will compute a skeleton polygon to "
                 + "support cases where camera window size is bigger than some regions of the "
                 + "confining polygon.  Enable only if needed, because it's costly");
         GUIContent m_MaxWindowSizeLabel;
-        GUIContent m_InvalidateCacheLabel = new GUIContent(
-            "Invalidate Cache", "Force a recomputation of the polygon cache.  "
-                + "This needs to be done if points inside the bounding polygon change");
+        GUIContent m_InvalidateComputedConfinerLabel = new(
+            "Invalidate Computed Confiner", "Invalidates the current confiner, so a new one is computed next frame.\n" +
+            "Call this when when the Field of View or Orthographic Size changes!");
+        GUIContent m_InvalidateFullCacheLabel = new(
+            "Invalidate Bounding Shape Cache", 
+            "Forces a re-computation of the whole confiner2D cache next frame.  This recomputes:\n" +
```

Comment:
This is better, but it's still not doing it for me.  Asks the naive user who has not looked at the implementation: "what is the difference between the current confiner and the whole confiner?"

There should be 2 invalidation commands:
1. InvalidatePolygonCache()
2. InvalidateLensCache()
You call the first one when the polygon changes, you call the second one when the lens changes.
You should use these names, and update the tooltips, docs, method and variable names accordingly.

Reply:
Good names :)